### PR TITLE
Preparing release 4.1.3

### PR DESCRIPTION
**Pull request by vegaro**
Should fix https://github.com/RevenueCat/react-native-purchases/issues/255

Looks like release 4.1.2 missed the update on the podspec. This release fixes that

### CODE REVIEW

**Comment by aboedo** [comment]
you have to clean up the `Podfile.lock`, or just run `pod update Purchases`, to fix the tests. I'm actually going to do a quick release of hybrid common for https://github.com/RevenueCat/purchases-hybrid-common/pull/78 to fix a crash on `syncPurchases`, could you hold and we can bump to 1.6.4 instead?

**Comment by aboedo** [comment]
here's the PR for the hybrid-common release https://github.com/RevenueCat/purchases-hybrid-common/pull/79

**Comment by aboedo** [comment]
Update: `purchases-hybrid-common` 1.7.0 released

**Comment by beylmk** [comment]
@vegaro reminder to update to hybrid-common 1.7.1 now!

**Review by beylmk — APPROVED** [review]
👍 want to keep this separate from canMakePayments?

**Comment by vegaro** [comment]
yeah I can just release this quickly since it is ready

File: android/build.gradle [hunk 1]
```diff
@@ -122,6 +122,6 @@ def kotlin_version = getExtOrDefault('kotlinVersion')
 dependencies {
     //noinspection GradleDynamicVersion
     api 'com.facebook.react:react-native:+'
-    implementation 'com.revenuecat.purchases:purchases-hybrid-common:1.6.3'
+    implementation 'com.revenuecat.purchases:purchases-hybrid-common:1.7.0'
```
Comment (beylmk) [inline android/build.gradle:125]: eeek just noticed this one isn't up to 1.7.1. i can include with canmakepayments?

Comment (vegaro) [reply android/build.gradle:125]: ugh its the second time this happens to us... Ill do a patch since canMakePayments is a minor right?

Comment (beylmk) [reply android/build.gradle:125]: good point, yeah it is as minor. maybe we could add pre-commit hook to check that ios and android are the same? i can look into this in my automation ticket

Comment (vegaro) [reply android/build.gradle:125]: I am actually going to write a quick script to increase all these versions so we dont miss any file

**Pull request merged**